cmd/prysmctl/db/query.go

```diff
@@ -0,0 +1,97 @@
+package db
+
+import (
+	"bytes"
+	"fmt"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/pkg/errors"
+	"github.com/prysmaticlabs/prysm/v3/config/params"
+	log "github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+	bolt "go.etcd.io/bbolt"
+)
+
+var queryFlags = struct {
+	Path     string
+	Bucket   string
+	KeysOnly bool
+	Prefix   string
+}{}
+
+var queryCmd = &cli.Command{
+	Name:   "query",
+	Usage:  "database query tool",
+	Action: queryAction,
+	Flags: []cli.Flag{
```

Comment:
Things I would change:
1. Make `path` mandatory and have it as the first flag.
2. Make `bucket` mandatory.
3. Change `print-keys` to `keys-only`. After all, without `print-keys` keys are still printed.